beacon_node/operation_pool/src/attestation.rs

```diff
@@ -119,14 +119,14 @@ impl<'a, E: EthSpec> AttMaxCover<'a, E> {
 
 impl<'a, E: EthSpec> MaxCover for AttMaxCover<'a, E> {
     type Object = Attestation<E>;
-    type Intermediate = AttestationRef<'a, E>;
+    type Intermediate = CompactAttestationRef<'a, E>;
     type Set = HashMap<u64, u64>;
 
-    fn intermediate(&self) -> &AttestationRef<'a, E> {
+    fn intermediate(&self) -> &CompactAttestationRef<'a, E> {
         &self.att
```

Comment:
This should return `CompactAttestationRef` rather than `&CompactAttestationRef` (assuming the `CompactAttestationRef` type is made by `superstruct`

Reply:
nevermind.. looks like it's not